### PR TITLE
Upload files to S3

### PR DESCRIPTION
**Pull request by AlexeyRaga**
Function to upload to S3

### CODE REVIEW

File: src/App/Conduit/Time.hs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import App.Types             (Seconds (..))
+import Conduit
+import Data.Time.Clock       (UTCTime)
+import Data.Time.Clock.POSIX (getCurrentTime, utcTimeToPOSIXSeconds)
+
+-- | Annotates every message with a timestamp.
+timedC :: MonadIO m
+       => Conduit a m (UTCTime, a)
+timedC = awaitForever $ \a ->
+  yieldM ((,a) <$> liftIO getCurrentTime)
```
Comment (newhoggy) [inline src/App/Conduit/Time.hs:14]: What's this for?

Comment (AlexeyRaga) [reply src/App/Conduit/Time.hs:14]: It annotates every message with a time

Comment (newhoggy) [reply src/App/Conduit/Time.hs:14]: I mean, I don't see it used.

Comment (AlexeyRaga) [reply src/App/Conduit/Time.hs:14]: It is not yet used

File: src/App/Conduit/Time.hs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+sampleC (Seconds sec) = go 0
+  where
+    go t = do
+      mmsg <- await
+      case mmsg of
+        Nothing -> pure ()
+        Just msg -> do
+          time <- liftIO getCurrentTime
+          case round (utcTimeToPOSIXSeconds time) of
+            ct | t == 0 -> go (ct + sec)                      -- initial, returse to the next interval
```
Comment (newhoggy) [inline src/App/Conduit/Time.hs]: Do you mean `recurse`? 😁

Comment (AlexeyRaga) [reply src/App/Conduit/Time.hs]: No, I meant `recurse` :)

**Comment by newhoggy** [comment]
👍

**Pull request merged**